scrape_data/scrape_schedule_versions.py

```diff
@@ -14,6 +14,36 @@
 BASE_URL = "https://transitfeeds.com"
 
 
+class ScheduleFeedInfo:
+    """Represents a single schedule version with feed start and end dates.
+    """
+
+    def __init__(self, version, start_date, end_date):
+        self.schedule_version = version
+        self.feed_start_date = start_date
+        self.feed_end_date = end_date
+
+    def __str__(self):
+        return f'v_{self.schedule_version}_fs_{self.feed_start_date}_fe_{self.feed_end_date}'
+
+    def __getitem__(self, item):
+        if item not in frozenset(['schedule_version', 'feed_start_date', 'feed_end_date']):
+            raise KeyError(item)
+        return self.__dict__[item]
+
+    @classmethod
+    def from_pendulum(cls, version, start_date, end_date):
+        return cls(version.format("YYYYMMDD"),
+                   start_date.format("YYYY-MM-DD"),
+                   end_date.format("YYYY-MM-DD"))
+
+    @classmethod
+    def from_dict(cls, d):
```

Comment:
Can this method be deleted since it's not used anywhere else?

Reply:
If there is only one method in this class, would it be better as a `namedtuple` or `dataclass`?

Reply:
Yes, `dataclass` is better and unused method can be removed. Updated both.